Themes/_fallback/Scripts/03 CustomSpeedMods.lua

```diff
@@ -70,6 +70,8 @@ end
 -- Parses a speed mod and returns the pair (type, number) or nil if parsing
 -- failed.
 local function CanonicalizeMod(mod)
+	local num = nil
```

Comment:
this should be dropped in favor of just local num = tonumber(...) on next line

Reply:
Yeah, good call.  Updated and force-pushed.